core/src/mindustry/entities/comp/BuilderComp.java

```diff
@@ -332,7 +333,7 @@ public void drawBuildingBeam(float px, float py){
         Lines.stroke(1f, plan.breaking ? Pal.remove : Pal.accent);
         Draw.z(Layer.buildBeam);
 
-        Draw.alpha(buildAlpha);
+        Draw.alpha(buildAlpha * Renderer.unitLaserOpacity);
```

Comment:
Should it be applied to build beams?

Reply:
I don't think build beams are very intrusive, because they don't stack, and you can still see under them. But it's a matter of opinion.